.travis.yml

```diff
@@ -239,7 +207,7 @@ matrix:
       - CAMLP5_VER=".6.17"
       - NATIVE_COMP="no"
       - COQ_DEST="-prefix ${PWD}/_install"
-      - EXTRA_CONF="-coqide opt -warn-error"
+      - EXTRA_CONF="-coqide opt -warn-error yes"
```

Comment:
That's the packaging job. It is not run on PRs. Why don't you put this extra flag in the job above instead?

Reply:
I didn't change anything here, you're looking at a commit from another PR.

Reply:
Right, sorry.